persist/sqlite/consensus.go

```diff
@@ -325,25 +325,25 @@ func addTransactionFields(tx *txn, txns []types.Transaction, txnExist map[types.
 		}
 
 		if err := addMinerFees(tx, dbID, txn.MinerFees); err != nil {
-			return fmt.Errorf("failed to add miner fees: %w", err)
+			return fmt.Errorf("failed to add miner fees: addMinerFees: %w", err)
```

Comment:
`addMinerFees:` is unnecessary. `failed to add miner fees:` is unique enough context to track it down. Ditto for all of these.